docs/releases/1.9.txt

```diff
@@ -65,6 +65,15 @@ the included auth forms for your project, you could set, for example::
 
 See :ref:`password-validation` for more details.
 
+Permission mixins for class-based views
```

Comment:
For me, major features are those that provide major new things or make the life much easier for many people.